draft-ietf-quic-transport.md

```diff
@@ -640,19 +584,23 @@ STREAM_DATA_BLOCKED ({{frame-stream-data-blocked}}), and RST_STREAM
 
 A sender MUST NOT send any of these frames from a terminal state ("Data Recvd"
 or "Reset Recvd").  A sender MUST NOT send STREAM or STREAM_DATA_BLOCKED after
-sending a RST_STREAM; that is, in the "Reset Sent" state in addition to the
-terminal states.  A receiver could receive any of these frames in any state, but
-only due to the possibility of delayed delivery of packets carrying them.
+sending a RST_STREAM; that is, in the terminal states and in the "Reset Sent"
+state.
+
+A receiver could receive any of these three frames in any state, due to the
```

Comment:
"these three frames" refers to something in another paragraph, so I think that your split should be reverted.